src/gui/mainwindow.cpp

```diff
@@ -1218,15 +1227,6 @@ void MainWindow::closeEvent(QCloseEvent *e)
         }
     }
 
-    // abort search if any
-    delete m_searchWidget;
-
-    hide();
```

Comment:
About removing all the related code:
Wasn't the original intention to hide the UI as quickly as possible to avoid unnecessary user interaction with the application shutting down?

Reply:
Yes, I've pushed another update that will disable the menu and show appropriate tooltips instead of hiding it immediately.
IMO hiding it immediately gives a false impression that qbt has already exited while it is still doing other clean ups.